examples/cluster/plot_bisect_kmeans.py

```diff
@@ -6,9 +6,13 @@
 This example shows differences between Regular K-Means algorithm and Bisecting K-Means.
 
 While K-Means clusterings are different when with increasing n_clusters,
-Bisecting K-Means clustering build on top of the previous ones.
+Bisecting K-Means clustering builds on top of the previous ones. As a
+result, it tends to create clusters that have a more consistent
+large-scale structure across the clusters.
 
-This difference can visually be observed.
+This difference can visually be observed: for all numbers of clusters,
+there is a dividing line cutting the overall data cloud in two for
+BisectingKMeans, but not for regular KMeans.
```

Comment:
```suggestion
dividing line cutting the overall data cloud in two for BisectingKMeans, while
regular KMeans results into a `Voronoi cell partitioning
<https://en.wikipedia.org/wiki/Voronoi_cell>`_.
```

Reply:
If you agree, I'll remove the comment on Voronoi cells. I'm not sure that it is necessary to interpret this example.

Reply:
Sure, feel free to do so. It is only a suggestion.